### PR TITLE
Fix #7425 prefix pluginIdentity id with 'app/' to check permission ( a…

### LINKED ISSUE #7425: Can not give permissions to view app
### How to Reproduce
- Sign in as admin
- Add app via app manager ( for example biobank-plots  [Archive.zip] )(https://github.com/molgenis/molgenis/files/2076602/Archive.zip)
- Add the app to the menu 
- Give rights to user ( anonymous ) to view the app ( rights 'app' plugin , rights to app itself, right to system entities )
- Sign out 
- Select the app from the menu

### Expected behavior
The user can now view the app

### Observed behavior
The use can not view the app and is shown a unstyled html page with the following message:

"401

No 'View plugin' permission on plugin with id 'biobank-plots'. (DP01)"

`org.molgenis.data.plugin.model.PluginPermissionDeniedException: pluginPermission: VIEW_PLUGIN, pluginId:biobank-plots at org.molgenis.app.manager.controller.AppController.serveApp(AppController.java:69)`

### PR DESCRIPTION
**Pull request by ConnorStroomberg**
Fix 7425 prefix pluginIdentity id with 'app/' to check permission ( app id has this as prefix)

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: molgenis-app-manager/src/main/java/org/molgenis/app/manager/controller/AppController.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ public AppController(AppManagerService appManagerService, UserPermissionEvaluato
 	public ModelAndView serveApp(@PathVariable String pluginId, Model model, HttpServletRequest request,
 			HttpServletResponse response) throws IOException
 	{
-		PluginIdentity pluginIdentity = new PluginIdentity(pluginId);
+		PluginIdentity pluginIdentity = new PluginIdentity("app/" + pluginId + "/");
```
Comment (fdlk) [inline molgenis-app-manager/src/main/java/org/molgenis/app/manager/controller/AppController.java]: variable name pluginId is too confusing here
app/{uri}/ = pluginid

File: molgenis-app-manager/src/test/java/org/molgenis/app/manager/controller/AppControllerTest.java [hunk 2]
```diff
@@ -136,14 +136,14 @@ public void beforeMethod() throws URISyntaxException
 	@AfterMethod
 	public void afterMethod()
 	{
-		PluginIdentity pluginIdentity = new PluginIdentity("uri");
+		PluginIdentity pluginIdentity = new PluginIdentity("app/uri");
```
[the remark this answers is not in the document]
Comment (ConnorStroomberg) [reply molgenis-app-manager/src/test/java/org/molgenis/app/manager/controller/AppControllerTest.java]: removed method , every test sets up this mock

File: molgenis-app-manager/src/test/java/org/molgenis/app/manager/controller/AppControllerTest.java [hunk 3]
```diff
@@ -155,26 +155,28 @@ public void testServeApp() throws Exception
...
 	public void testServeAppInactiveApp() throws Exception
 	{
-		PluginIdentity pluginIdentity = new PluginIdentity("uri");
+		PluginIdentity pluginIdentity = new PluginIdentity("app/uri/");
 		when(userPermissionEvaluator.hasPermission(pluginIdentity, VIEW_PLUGIN)).thenReturn(true);
 
 		App app = mock(App.class);
 		when(app.getId()).thenReturn("id");
-		when(app.getUri()).thenReturn("uri");
+		when(app.getUri()).thenReturn("app/uri");
```
[the remark this answers is not in the document]
Comment (ConnorStroomberg) [reply molgenis-app-manager/src/test/java/org/molgenis/app/manager/controller/AppControllerTest.java]: done , it's not used in this test

File: molgenis-app-manager/src/main/java/org/molgenis/app/manager/controller/AppController.java (hunk 1 above)
Comment (ConnorStroomberg) [reply molgenis-app-manager/src/main/java/org/molgenis/app/manager/controller/AppController.java]: changed to appName

**Comment by fdlk** [comment]
test this please

**Comment by fdlk** [comment]
test this please

**Comment by fdlk** [comment]
test this please

**Pull request merged**